platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -253,7 +270,7 @@ - (void)pickFeatureAt:(CGPoint)screenPosition
     screenPosition.x *= self.contentScaleFactor;
     screenPosition.y *= self.contentScaleFactor;
 
-    self.map->pickFeatureAt(screenPosition.x, screenPosition.y, [screenPosition, self](const Tangram::FeaturePickResult* featureResult) {
+    self.map->pickFeatureAt(screenPosition.x, screenPosition.y, [=](const Tangram::FeaturePickResult* featureResult) {
```

Comment:
This does not copy _all_, only what's referenced from the lambda body, so `self`, to increment its counter in ARC context. 

Refer https://github.com/tangrams/tangram-es/pull/1347#discussion_r107471131.